src/dosbox.cpp

```diff
@@ -1135,6 +1135,11 @@ void DOSBOX_Init()
 	        "(auto by default, enabled if DOS version >= 7.0).\n"
 	        "FreeDOS and MS-DOS 7/8 COMMAND.COM supports this behavior.");
 
+	pbool = secprop->Add_bool("shell_config_shortcut", when_idle, true);
```

Comment:
Suggest plural: `allow_shell_config_shortcuts = yes/no`

Reply:
Actually, that's a lot better, yes.